vscode/quint-vscode/server/README.md

```diff
@@ -0,0 +1,22 @@
+# Quint Language Server
+
+[![npm (scoped)](https://img.shields.io/npm/v/@informalsystems/quint-language-server)](https://www.npmjs.com/package/@informalsystems/quint-language-server)
```

Comment:
This link looks like two links?

Reply:
It's the badge thing, I believe the first one gets the image and the second is the actual link that opens when you click the badge. I just copied from quint's readme and changed the links.